mozregression/releases.py

```diff
@@ -63,13 +67,36 @@ def releases():
         53: "2017-01-23",
         54: "2017-03-06",
         55: "2017-06-12",
-        56: "2017-08-02",
-        57: "2017-09-21",
-        58: "2017-11-13",
-        59: "2018-01-22",
-        60: "2018-03-12"
+        56: "2017-08-02"
     }
 
+    def filter_tags(tag_node):
+        RE_TAG = re.compile("^FIREFOX_NIGHTLY_(\d+)_END$")
+        match = RE_TAG.match(tag_node["tag"])
```

Comment:
Probably no need to define a once-off regular expression here. Just:

```py
match = re.match("^FIREFOX_NIGHTLY_(\d+)_END$", tag_node["tag"])
```

should suffice.